.readthedocs.yml

```diff
@@ -17,6 +17,8 @@ sphinx:
   fail_on_warning: true
 
 python:
-   install:
-   - method: pip
-     path: .
+  install:
+  - method: pip
+    path: .
+    extra_requirements:
+      - sphinx_rtd_theme
```

Comment:
should it be `sphinx-rtd-theme` ?

Reply:
Actually this translated to `pip install .[sphinx_rtd_theme]` (sorry the docs were not clear...). So I think the solution is to change this with `docs` and add `sphinx-rtd-theme` into `extra_requires['docs']`.